lib/Backend/arm64/LowerMD.cpp

```diff
@@ -5796,7 +5797,7 @@ bool LowererMD::GenerateFastCharAt(Js::BuiltinFunction index, IR::Opnd *dst, IR:
     }
     // LDRH charReg, [r1 + r2 * 2]  -- this is the value of the char
     IR::RegOpnd *charReg = IR::RegOpnd::New(TyMachReg, this->m_func);
-    instr = IR::Instr::New(Js::OpCode::LDR, charReg, indirOpnd, this->m_func);
+    instr = IR::Instr::New(Js::OpCode::LDRS, charReg, indirOpnd, this->m_func);
```

Comment:
>LDRS [](start = 39, length = 4)

why is LDRS needed here?

Reply:
indirOpnd here is TyInt16. This could be handled through legalize though.